packages/governance/src/tests/test_fuzz_packing.cairo

```diff
@@ -0,0 +1,38 @@
+use crate::governor::ProposalCore;
```

Comment:
We already have this file now in main, so we need to merge these tests into that.